src/CommentSold/Api/GlobalClient.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api;
+
+use CommentSold\Api\Clients\Rest;
+
+class GlobalClient
+{
+    private Rest $restClient;
+
+    public function __construct(private readonly string $partnerToken)
+    {
+        $this->restClient = new Rest();
+    }
+
+    /**
+     * Get tax quote for cart
+     */
+    public function getTaxQuote(array $payload)
+    {
+        return $this->restClient->post($this->partnerToken, 'quote', $payload);
+    }
+
+    /**
+     * Create Shop
+     */
+    public function createShop(array $payload)
+    {
+        return $this->restClient->post($this->partnerToken, 'accounts', $payload);
+    }
+
+    /**
+     * Request a URL to redirect customers to prompt them for OAuth authorization
+     */
+    public function getOauthUrl(array $scopes, string $redirectUrl)
+    {
+        return $this->restClient->get($this->partnerToken, 'accounts/authorizeUrl?scopes='.implode(',', $scopes).'&redirect_uri='.$redirectUrl);
+    }
+
+    /**
+     * Get sub categories of category from optional parent category ID
+     */
+    public function getSubCategories(int $categoryId = 0, $page = 1, $perPage = Rest::PER_PAGE)
```

Comment:
```suggestion
    public function getSubCategories(int $categoryId = 0, int $page = 1, int $perPage = Rest::PER_PAGE)
```
For a public API, I'd also validate that those ints are `> 0`